allennlp_models/modelcards/nlvr2-vilbert-head.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
I'm not sure it makes sense to have a separate model card for the head

Reply:
I'm not sure either, but I talked with @dirkgr about it and we settled on two iirc.

Reply:
It makes some sense to me, given that it's a separate model file with a separate training config, even if the results are the same.